vendor/QmXJkcEXB6C9h6Ytb6rrUTFU56Ro62zxgrbxTT3dgjQGA8/go-log/oldlog.go

```diff
@@ -1,6 +1,7 @@
-package util
+package log
 
 import (
+	"errors"
 	"os"
 
 	"github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/Sirupsen/logrus"
```

Comment:
Should this be moved to go-log's Godeps and that `github.com/ipfs/go-ipfs/thirdparty/pollEndpoint` uses go-log as well?